src/views/EdgeApplicationsCacheSettings/FormFields/FormFieldsEdgeApplicationCacheSettings.vue

```diff
@@ -188,12 +188,14 @@
       cdnCacheSettings.value = 'override'
       isSliceEdgeCachingEnabled.value = true
       sliceConfigurationEnabled.value = true
-      emit('l2-caching-enabled', value)
     } else {
+      if (!props.isEnableApplicationAccelerator && cdnCacheSettingsMaximumTtl.value < 60) {
```

Comment:
Vamos evitar o encadeamento de condicionais, torna dificil a leitura do código.

Sugiro algo assim:
```suggestion
	const isNotApplicationAccelerator = !props.isEnableApplicationAccelerator && cdnCacheSettingsMaximumTtl.value < 60
	
      if (!value && isNotApplicationAccelerator) {
      // Código
      }
      
      if (value){
      // Código
      }
```